action/entrypoint.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -eu
+
+SOURCE_DIR="$(dirname -- ${BASH_SOURCE[0]:-$0})"
+
+log() {
+    echo -e "$1" >&2
+}
+
+_DEFAULT_INSTALL_DIR=${HOME}/bin
+_INSTALL_DIR=${INSTALL_DIR:-${_DEFAULT_INSTALL_DIR}}
+CMD_NAME="taplo"
+COMMAND="${_INSTALL_DIR}/${CMD_NAME}"
+
+TARGET=${INPUT_FILES:-"."}
+if [[ -z $(ls ${TARGET} 2>/dev/null) ]]; then
+    log "ERROR: Input files (${TARGET}) not found"
+    exit 1
+fi
+
+if [[ ! -x ${COMMAND} ]]; then
+    VERSION=${INPUT_VERSION:-"0.9.3"}
+    if [[ "$(uname -m)" == "arm64" || "$(uname -m)" == "aarch64" ]]; then
+        ARCH="aarch64"
+    else
+        ARCH="x86_64"
+    fi
+    UNAME=$(uname -s)
+    if [[ "$UNAME" == "Darwin" ]]; then
+        TARGET_FILE="full-darwin-${ARCH}"
+        FILE_EXT="gz"
+    elif [[ "$UNAME" == CYGWIN* || "$UNAME" == MINGW* || "$UNAME" == MSYS* ]]; then
+        TARGET_FILE="windows-${ARCH}"
+        FILE_EXT="zip"
+    else
+        TARGET_FILE="linux-${ARCH}"
+        FILE_EXT="gz"
+    fi
+    FILE_NAME="${CMD_NAME}-${TARGET_FILE}.${FILE_EXT}"
+    log "Downloading '${CMD_NAME}' v${VERSION}"
+    wget --progress=dot:mega "https://github.com/tamasfe/taplo/releases/download/${VERSION}/${FILE_NAME}"
+
+    mkdir -p ${_INSTALL_DIR}
+    if [[ "$FILE_EXT" == "zip" ]]; then
+        unzip -o "${FILE_NAME}" -d ${_INSTALL_DIR} ${CMD_NAME}.exe
+    else
+        gzip -d "${FILE_NAME}" -c >${COMMAND}
+        chmod +x ${COMMAND}
+    fi
+    rm "${FILE_NAME}"
+fi
+log "jq: $(jq --version)"
+
+ARGS="format"
```

Comment:
Would be great to also have support for `taplo check`. And perhaps rename `format` to `fmt` (at least the docs seem to prefer fmt).

Reply:
as `fmt` is an alias for `format`, it's seems most logical to me to use `format` unless `fmt` presented as preferred way ?
I haven't explored the `check`/`lint` functionality, what it the interest in CI ? What you think it's should do ? How should it be integrated ?

Reply:
I cannot say anything about the preference of `fmt` vs `format` except that the docs seem to only mention `fmt`

The check would work similarly as for `fmt`: run `taplo check`, if it reports a problem fail. So my proposal would be to add two boolean toggles (both defaulting to true?), each for enabling/disabling the the fmt and check run.

Reply:
Ok, thank.